src/main/java/uk/ac/ebi/atlas/hcalandingpage/HcaMetadataService.java

```diff
@@ -0,0 +1,78 @@
+package uk.ac.ebi.atlas.hcalandingpage;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.gson.JsonObject;
+import org.springframework.stereotype.Component;
+import uk.ac.ebi.atlas.experiments.ExperimentJsonSerializer;
+import uk.ac.ebi.atlas.model.experiment.Experiment;
+import uk.ac.ebi.atlas.trader.ExperimentTrader;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+
+@Component
+public class HcaMetadataService {
+    private final HcaMetadataDao hcaMetadataDao;
+    private final ExperimentTrader experimentTrader;
+
+    public HcaMetadataService(ExperimentTrader experimentTrader,
+                              HcaMetadataDao hcaMetadataDao) {
+        this.experimentTrader = experimentTrader;
+        this.hcaMetadataDao = hcaMetadataDao;
+    }
+
+    public List<String> getHcaOntologyIds() {
+        return getMetadata().get("ontology_ids");
+    }
```

Comment:
Good point! order of ids doesn't matter but I don't want duplicates so have instead used a `Set` instead of the list.

Okay! will keep that in mind.